pxr/usd/plugin/usdAbc/alembicReader.cpp

```diff
@@ -894,22 +878,58 @@ _ReaderContext::_ReaderContext() :
 }
 
 bool
-_ReaderContext::Open(const std::string& filePath, std::string* errorLog)
+_ReaderContext::Open(const std::string& filePath, std::string* errorLog,
+                     const SdfFileFormat::FileFormatArguments* args)
 {
     Close();
 
-    IArchive archive;
+    std::vector<std::string> layeredABC;
+    if (args) {
+        const auto abcLayers = args->find("abcLayers");
+        if (abcLayers != args->end()) {
+            for (auto&& l : TfStringSplit(abcLayers->second, ",")) {
+                layeredABC.emplace_back(std::move(l));
```

Comment:
I know Alembic doesn't do any asset resolution, by-design... but would it not be valuable in the USD universe to attempt to anchor (to "filePath") and resolve each of these paths using the installed ArAssetResolver before passing the paths down to Alembic?

Reply:
I've looked into this, but it's currently a bit problematic on the Alembic side to pass layers as relative (particularly in a multi-threaded env).  Second, there may be cases where the layer is in a totally separate location, unrelated to the -base layer-.

Or are you suggesting to call the resolver within usdAbc.so ?

Reply:
I wasn't suggesting that you pass relative paths to Alembic.  Instead, use ArGetResolver().Resolve() (etc) to resolve the paths to full paths **in USD-land** before passing them on to Alembic.

But if that seems undesirable, then just please document in overview.dox that you **must** only supply full paths in the URI, which is a bit limiting for fancy resolver setups, but probably OK since this is for Alembic backwards compatibility.

Thanks!

Reply:
Ah OK, then yes (i.e. this is currently already being done on the resolver side here), I'll add the note.